manifests/params.pp

```diff
@@ -36,6 +36,11 @@
   $rdeck_home = '/var/lib/rundeck'
   $service_logs_dir = '/var/log/rundeck'
 
+  $rdeck_uuid = $::serialnumber ? {
+    undef   => fqdn_uuid($::fqdn),
+    default => $::serialnumber,
```

Comment:
So it might be worth doing at least?
```puppet
$rdeck_uuid = $facts['serialnumber'] ? {
  undef   => fqdn_uuid($::fqdn),
  '0'     => fqdn_uuid($::fqdn),
  default => $::serialnumber,
}
```